% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/DebugFn.R
\name{returnCapture}
\alias{returnCapture}
\title{Return an error to a file, environment (no names) or callback}
\usage{
returnCapture(
  e,
  saveDest,
  cap,
  wrapperName,
  recallString = "do.call(p$fn, p$args)"
)
}
\arguments{
\item{e}{caught exception}

\item{saveDest}{where to save}

\item{cap}{saved arguments}

\item{wrapperName}{name of wrapper}

\item{recallString}{how to call function again}
}
\description{
Return an error to a file, environment (no names) or callback
}
\keyword{internal}
